pypeit/reduce.py

```diff
@@ -412,13 +424,17 @@ def run(self, basename=None, ra=None, dec=None, obstime=None,
         # Finish up
         if self.sobjs.nobj == 0:
             msgs.warn('No objects to extract!')
-        else:
+        elif self.par['flexure']['spec_method'] not in ['skip', 'slit_cen']:
+            # Apply a refined estimate of the flexure to objects, and then apply reference frame correction to objects
             # TODO -- Should we move these to redux.run()?
             # Flexure correction if this is not a standard star
-            if not self.std_redux:
-                self.spec_flexure_correct(self.sobjs, basename)
-            # Heliocentric
-            self.helio_correct(self.sobjs, ra, dec, obstime)
+            if (not self.std_redux):
```

Comment:
Why the added paranthesis here, I think: 

if not self.std_redux 

works just fine.